First Update README.md (#26)
alckasoc/discussion-agents: README.md

# discussion-agents

Discussion agents. 

# Developer Setup: Windows

First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and add to path by going to advanced system settings. Then launch cmd.

Use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
```
conda create -n discussion-agents python=3.10.13
```
Now activate the environent.
```
conda activate discussion-agents
```

> Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/)

Reviewer: ```suggestion
Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/).
```

```
pip install pipx
pipx install poetry
```

> Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.

Reviewer: ```suggestion
Make sure to add poetry to path by adding `C:\Users\<username>\.local\bin` to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
```

Then clone the repository and enter the discussion-agents directory.
``` 

> git clone https://github.com/alckasoc/discussion-agents/
> 

Reviewer: ```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

> git clone https://github.com/alckasoc/discussion-agents/
> 

Reviewer: ```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

# discussion-agents

Discussion agents. 

> # Developer Setup: Windows

Reviewer: ```suggestion
# Developer Setup
```

1. First install [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/) and add to path by going to advanced system settings. Then launch cmd.

> Use the following command to create a conda environment named my_env with python version 3.10.13.

Reviewer: ```suggestion
Use the following command to create a conda environment `discussion-agents` with Python version 3.10.13. Any Python version above 3.9 is viable.
```

```

> 2. conda create -n my_env python=3.10.13

Reviewer: ```suggestion
conda create -n my_env python=3.10.13
```

```
Now activate the environent.
```

> 3. conda activate my_env

Reviewer: ```suggestion
conda activate my_env
```

```

> Next install [pipx](https://pipx.pypa.io/stable/docs/) to install [poetry](https://python-poetry.org/docs/).

Reviewer: ```suggestion
Next, we will install [Poetry](https://python-poetry.org/docs/) using [pipx](https://pipx.pypa.io/stable/docs/).
```

```

> 4. pip install pipx
> 5. pipx install poetry

Reviewer: ```suggestion
pip install pipx
pipx install poetry
```

> 4. pip install pipx
> 5. pipx install poetry

Reviewer: ```suggestion
pip install pipx
pipx install poetry
```

```

> Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings.

Reviewer: ```suggestion
Make sure to add poetry to path by adding C:\Users\<username>\.local\bin to path in advanced system settings. For other operating systems, the path will be different. Ensure poetry is in the environment variable paths.
```

Then clone the repository and enter the discussion-agents directory.
``` 

> 6. git clone https://github.com/alckasoc/discussion-agents/

Reviewer: ```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

> 6. git clone https://github.com/alckasoc/discussion-agents/

Reviewer: ```suggestion
git clone https://github.com/alckasoc/discussion-agents/
```

```
Finally install all of the packages.
```

> 7. poetry install

Reviewer: ```suggestion
poetry install
```

Discussion agents.

> # Windows installation 

Reviewer: Developer Setup: Windows

```

> 1.First install anaconda https://docs.anaconda.com/free/anaconda/install/windows/ and add to path, or launch anaconda console

Reviewer: Add a space after the number

Reviewer: Also, the link can be formatted with markdown like this [anaconda](https://docs.anaconda.com/free/anaconda/install/windows/). End each step with a period (full sentences).

Reviewer: Probably go into detail (alternatively, you can add a link to some resource for how to add stuff to the path) describing how to add to the path.

> 2.conda create -n my_env python=3.10.13

Reviewer: Code can go in the code block and the text can go outside of the code block. Maybe add a statement above the code block to describe the step, the command, and the flags. 

What is `conda create`?
What is `-n`?
What is the `my_env` argument?
What is the `python=3.10.13`?

3.conda activate my_env

> 4.pip install pipx

Reviewer: Maybe also add a link to pipx docs for easy reference.

> 5.pipx install poetry (make sure to add poetry to path by adding C:\Users\<username\.local\bin to path in advanced system settings)

Reviewer: Missing ">" sign after "username". Probably want to go into detail about adding the path.

> 6.In the discussion agents directory - poetry install

Reviewer: Maybe add specify how a developer can test that their environment is properly set up.

> ```

Reviewer: Newline
